.gitignore

```diff
@@ -328,3 +328,4 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+/src/Arcus.Security.Tests.Integration/appsettings.json
```

Comment:
Feel free to align with our Event Grid approach and provide a tokenized config https://github.com/arcus-azure/arcus.eventgrid/blob/master/src/Arcus.EventGrid.Tests.Integration/appsettings.json

Reply:
But I'm open for suggestions, how would you run the tests with this approach? Just rely on the environment variables?

The downside is that new people need to go through all tests to know what variables are required to run them locally.

Reply:
I agree it's not perfect and I thought I picked the same way of loading settings from the EventGrid integration tests.  Now, I just excluded the appsettings.json through gitignore, but people have to build their own way.  Will implement it in a new PR, coming up.

Reply:
created #36